packages/cli/src/commands/validatorgroup/commission.ts

```diff
@@ -7,18 +7,19 @@ import { Flags } from '../../utils/command'
 
 export default class ValidatorGroupCommission extends BaseCommand {
   static description =
-    'Manage the commission for a registered Validator Group. This represents the share of the epoch rewards given to elected Validators that goes to the group they are a member of.'
+    'Manage the commission for a registered Validator Group. This represents the share of the epoch rewards given to elected Validators that goes to the group they are a member of. Updates must be made in a two step process where the group owner first calls uses the queue-update option, then after the required update delay, the apply option. The commission update delay, in blocks, can be viewed with the network:parameters command. A groups next commission update block can be checked with validatorgroup:show'
 
   static flags = {
     ...BaseCommand.flags,
     from: Flags.address({ required: true, description: 'Address for the Validator Group' }),
```

Comment:
Probably best to change this one too
```suggestion
    from: Flags.address({ required: true, description: 'Address for the Validator Group or Validator Group validator signer' }),
```